tests/ci/cdk/README.md

```diff
@@ -85,6 +85,41 @@ For help, run command:
 ./run-cdk.sh --help
 ```
 
+## AWS-LC Benchmarking Framework
+
+### Framework Setup
+No special actions outside those outlined in the **CI Setup** section are required to deploy the benchmarking framework to an AWS account and GitHub repository.
+
+Specific implementation details can mostly be found in:
+* `run-cdk.sh`
+* `cdk/bm_framework_stack.py`
+
+### How to Use
+After the benchmarking framework is set up, there are multiple ways to start the process.
+
+Note: due to the nature of how the framework works, a commit containing the relevant scripts (located inside `tests/ci/benchmark_framework/`,`tests/ci/build_run_benchmarks.sh`, and `tests/ci/run_bm_framework.sh`) is required for a successful run of the framework. If these files are not present, the benchmarking process will fail.
+
+#### Start from Pull Request
+Opening, reopening, or updating a pull request in a repository in which the framework has been deployed to will start the benchmarking process automatically without any extra input needed from the user.
+
+#### Start Locally
+Starting the benchmarking framework locally still requires the user's AWS CLI to be configured for an account with the framework deployed to it. Additionally, several environment variables need to be set for the framework to work correctly.
+
+| Environment Variable      | Description            | Value                                                                         |
+|---------------------------|------------------------|-------------------------------------------------------------------------------|
+| `CODEBUILD_WEBHOOK_TRIGGER` | GitHub PR Number       | Can be any value when running locally, suggest leaving blank                  |
```

Comment:
If this can be set to anything, and is suggested to leave blank, do we need to list this here? The sentence above states that certain environment variables *need* to be set, and it sounds like this one is unnecessary.